### ISSUE TITLE
liberica-runtime-container lost arm64 version for jdk and jre tag

### ISSUE THREAD

**Issue by fangzhengjin** [bell-sw/Alpaquita#4]
![image](https://github.com/user-attachments/assets/4c45a447-adad-4ea6-8504-9c7c038ba241)

**Comment by fangzhengjin** [comment]
@voitylov The new version only has the jdk-all tag for both amd64 and arm64 architectures.

**Comment by voitylov** [comment]
I don't think we had crac builds published for AArch64 just yet. It's definitely on the roadmap.

**Comment by voitylov** [comment]
CRaC builds are now available for AArch64.
